### PR TITLE
vh-mob1830-remove-cvv

### PR DESCRIPTION
**Pull request by vhuynh**
http://jira/browse/MOB-1830

Update card assets.
Updated code to remove CVV.

### CODE REVIEW

File: library/src/main/java/com/paymentkit/views/FieldHolder.java [hunk 1]
```diff
@@ -99,100 +92,98 @@ public boolean isFieldsValid() {
...
         return true;
     }
 
-    /** this listener is a great place to call:
+    /**
+     * this listener is a great place to call:
      * ViewUtils.hideSoftKeyboard((Activity)getContext());
      * clearFocus();
-     *
+     * <p/>
```
Comment (mineshpatel1511) [inline library/src/main/java/com/paymentkit/views/FieldHolder.java]: Did you add this?

File: library/src/main/java/com/paymentkit/views/FieldHolder.java [hunk 2]
```diff
@@ -227,45 +217,40 @@ public void onAnimationEnd(Animator anim) {
      * @return true if it focuses on an invalid field.
      **/
     private boolean focusOnInvalidField() {
+        Log.w(TAG, "focusOnInvalidField");
```
Comment (mineshpatel1511) [inline library/src/main/java/com/paymentkit/views/FieldHolder.java]: You can take this out.

File: library/src/main/java/com/paymentkit/views/FieldHolder.java [hunk 3]
```diff
@@ -24,6 +25,7 @@
  * @author Brendan Weinstein
  */
 public class FieldHolder extends RelativeLayout {
+    private static final String TAG = FieldHolder.class.getSimpleName();
```
Comment (mineshpatel1511) [inline library/src/main/java/com/paymentkit/views/FieldHolder.java]: Do you need this or was just debugging?

File: library/src/main/res/layout/pk_field_holder.xml [hunk 4]
```diff
@@ -43,27 +43,10 @@
             android:hint="@string/pk_expiration_field_hint"
             android:imeOptions="actionNext|flagNoExtractUi"
             android:inputType="number"
-
             android:maxLength="5"
             android:textSize="@dimen/pk_checkout_input_field_text_size"
             />
 
-        <com.paymentkit.views.CVVEditText
```
Comment (mineshpatel1511) [inline library/src/main/res/layout/pk_field_holder.xml]: You can get rid of the enclosing LinearLayout and refactor this to lie outside.

**Comment by maherhanafi** [comment]
👍

File: library/src/main/java/com/paymentkit/views/FieldHolder.java (hunk 1 above)
Comment (vhuynh) [inline library/src/main/java/com/paymentkit/views/FieldHolder.java]: It comes when you hit auto format.

**Pull request merged**